### PR TITLE
Added density transition to EXPERIMENTAL_Table

### PR DESCRIPTION
**Pull request by matheusps**
#### What is the purpose of this pull request?

As the title says.

#### How should this be manually tested?
Clone the repo and `yarn & yarn start`

#### Screenshots or example usage
Before
before

After
Screen Recording 2019-11-04 at 09 46 55 2019-11-04 09_48_59

#### Types of changes

- [ ] Bug fix (a non-breaking change which fixes an issue)
- [x] New feature (a non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Requires change to documentation, which has been updated accordingly.

### CODE REVIEW

**Comment by mateusbolsoni** [comment]
I'm in doubt about the option names. I think Low, Medium and High it's a bit confusing and not intuitive. Here is how Google handle this with Gmail -> calling Default, Regular and Compact.

image

**Comment by matheusps** [comment]
We are discussing the densities renaming. This will be implemented on upcoming pull requests ☝🏾

File: react/components/EXPERIMENTAL_TableTree/README.md [hunk 1]
```diff
@@ -194,14 +194,16 @@ const columns = [
 const items = sampleData
 
 function Image({ size }) {
+  const style = {
+    width: '100%',
+    height: '100%',
```
Comment (kevinch) [inline react/components/EXPERIMENTAL_TableTree/README.md]: Could those two elements being replaced by `w-100` and `h-100` with css classes?

Comment (matheusps) [reply react/components/EXPERIMENTAL_TableTree/README.md]: yes, it could

**Comment by matheusps** [comment]
I've made the transition simpler. Wait for another PR

**Pull request closed without merging**